mage_ai/data_cleaner/shared/logger.py

```diff
@@ -20,4 +24,21 @@ def __exit__(self, type, value, traceback):
         # https://statsd.readthedocs.io/en/v3.1/timing.html
         dt = int((time.time() - self.start) * 1000)
         if self.verbose:
-            print(f'[time] metric: {self.metric}, value: {dt}ms, tags: {self.tags}')
+            logger.debug(f'[time] metric: {self.metric}, value: {dt}ms, tags: {self.tags}')
+
+
+class VerboseFunctionExec:
+    def __init__(self, message, verbose=True):
+        self.message = message
+        self.verbose = verbose
+
+    def __enter__(self):
+        if self.verbose:
+            print(f'{self.message}...', end='')
+
+    def __exit__(self, exc_type, exc_value, exc_tb):
```

Comment:
what's exc_tb used for?

Reply:
Gives the traceback for the exception that is intercepted while in the managed context. I didn't do anything with it since the exception will be propagated upwards to either be caught or crash the program.